design-documents/graph-ql/coverage/customer/gift-registry.graphqls

```diff
@@ -128,40 +130,63 @@ input GiftRegistryDynamicAttributeInput {
     value: String!
 }
 
-type CreateGiftRegistryOutput {
-    gift_registry: GiftRegistry
+interface GiftRegistryUserErrorsInterface {
+    status: Boolean!
+    user_errors: [GiftRegistryItemsUserError]!
+}
+
+interface GiftRegistryItemUserErrorInterface {
```

Comment:
Same comment about removing "User" from the name.

Reply:
I added a comment about, we would have to change it everywhere else but we also have to start somewhere